Doc'd how to use Intersphinx in the reusable apps tutorial. (#19297)
django/django: docs/intro/reusable-apps.txt

   you add some files to it. Many Django apps also provide their documentation
   online through sites like `readthedocs.org <https://readthedocs.org>`_.

   Many Python projects, including Django and Python itself, use `Sphinx
   <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ to build
   their documentation. If you choose to use Sphinx you can link back to the
   Django documentation using `Intersphinx
   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_

>    by including the a value for Django in your project's

Reviewer: ```suggestion
   by including a value for Django in your project's
```

   ``intersphinx_mapping`` configuration::

        intersphinx_mapping = {
            # ...
            "django": (

>                 "http://docs.djangoproject.com/en/stable/",
>                 "http://docs.djangoproject.com/en/stable/_objects/",

Reviewer: ```suggestion
                "https://docs.djangoproject.com/en/stable/",
                "https://docs.djangoproject.com/en/stable/_objects/",
```

(I did mention that one on the call before 😁 )

Author: Ah, yes. 🙌

   you add some files to it. Many Django apps also provide their documentation
   online through sites like `readthedocs.org <https://readthedocs.org>`_.

   Many Python projects, including Django and Python itself, use `Sphinx
   <https://www.sphinx-doc.org/en/master/usage/quickstart.html>`_ to build
   their documentation. If you choose to use Sphinx you can link back to the

>    Django documentation using `Intersphinx
>    <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
>    by including a value for Django in your project's ``intersphinx_mapping``
>    configuration::

Reviewer 2: ```suggestion
   Django documentation by configuring `Intersphinx
   <https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx>`_
   and including a value for Django in your project's ``intersphinx_mapping``
   value::
```
I'm not sure if this is clearer but folks would need to follow the instructions in order to configure Intersphinx (they need to add `'sphinx.ext.intersphinx'` to the `extensions` list and the `intersphinx_mapping` mapping value is within the Sphinx `conf.py`/Sphinx configuration file).
So hope this implies there are more steps in the process and this is within the Sphinx conf

Author: Yes! Thanks. That's much better.

        intersphinx_mapping = {
            # ...
            "django": (
                "https://docs.djangoproject.com/en/stable/",

>                 "https://docs.djangoproject.com/en/stable/_objects/",

Reviewer: I've merged https://github.com/django/djangoproject.com/pull/1995, so we can now use the default location of `objects.inv`:
```suggestion
                None,
```

Author: Great. Yes. 🙌
